test/test-server/connect_handler.go

```diff
@@ -71,7 +71,26 @@ func handleConnectors(t *testing.T) http.HandlerFunc {
 					Name:   connectv1.PtrString("az-connector"),
 				},
 			}
-			err := json.NewEncoder(w).Encode(map[string]connectv1.ConnectV1ConnectorExpansion{"az-connector": connectorExpansion})
+			secondConnectorExpansion := connectv1.ConnectV1ConnectorExpansion{
+				Id: &connectv1.ConnectV1ConnectorExpansionId{Id: connectv1.PtrString("lcc-111")},
+				Status: &connectv1.ConnectV1ConnectorExpansionStatus{
+					Name: "az-connector-2",
+					Connector: connectv1.ConnectV1ConnectorExpansionStatusConnector{
+						State: "RUNNING",
+						Trace: connectv1.PtrString(""),
```

Comment:
if it's empty can it be omitted?

Reply:
Yup! I wenet ahead and removed it from the block above, too.